src/python/CMSSpark/data_collection.py

```diff
@@ -45,8 +46,8 @@ def yesterday():
 
 def short_date_to_unix(date):
 
-    #Convert short date string into UNIX timestamp
-    return int(time.mktime(time.strptime(date, '%Y/%m/%d')))
+    #Convert short date string into UNIX timestamp (GMT)
+    return int(calendar.timegm(time.strptime(date, '%Y/%m/%d'))) * 1000
```

Comment:
Could you please add additional comment that returning time is in miliseconds and a reason why do you need that (i.e. for Kibana plotting).